latest/PIT/pit_common.yaml

```
@@ -0,0 +1,188 @@
+        description: |
+          The amount of time elapsed between the start and the end of
+          the observation or data stream.
+        type: number
+        unit: "s"
+      effective_exposure_time:
+        title: Effective exposure time (s)
+        description: |
+          The total accumulated effective exposure duration for this
+          data product, excluding dead time and lost time.
+        type: number
+        unit: "s"
+      instrument:
+        title: Instrument observing configuration
+        type: object
+        properties:
+          name:
+            title: Instrument name
+            anyOf:
+              - type: string
+              - type: "null"
+          detector:
+            title: Detector
+            anyOf:
+              - type: string
+              - type: "null"
+          optical_element:
+            title: Optical element
+            description: |
+              Name of filter, disperser, or other optical element.
+            anyOf:
+              - type: string
+              - type: "null"
+        required: [name, detector, optical_element]
+      telescope:
+        title: Telescope name
+        description: |
+          Name of telescope, typically Roman.
+        anyOf:
+          - type: string
+          - type: "null"
+      target_coordinates:
+        title: Position and region for this product
+        type: object
+        properties:
+          reference_frame:
+            anyOf:
+              - type: string
+              - type: "null"
+          ra:
+            title: Right ascension (deg)
+            description: |
+              Characteristic right ascension in degrees; typically the
+              coordinate of the source of interest, or the center of
+              the spatial image. May be null for large survey images,
+              simulated data, etc.
+            anyOf:
+              - type: number
+              - type: "null"
+            unit: "deg"
+          dec:
+            title: Declination (deg)
+            description: |
+              Characteristic declination in degrees; typically the
+              coordinate of the source of interest, or the center of
+              the spatial image. May be null for large survey images,
+              simulated data, etc.
+            anyOf:
+              - type: number
+              - type: "null"
+            unit: "deg"
+          s_region:
+            title: STC-S formatted sky footprint
+            description: |
+              Sky footprint for this data product, formatted as an
+              IVOA Space-Time Coordinate Metadata Linear String
+              (STC-S) in units of degrees.
+            anyOf:
+              - type: string
+              - type: "null"
+        required: [reference_frame, ra, dec, s_region]
+      pixel_scale:
+        # Optional
+        title: Pixel scale of spatial axes (arcsec/pixel)
+        type: number
+        unit: "arcsec/pixel"
+      spatial_dimensions:
```

```suggestion
      spatial_dimensions:
```
Is this equivalent to the `wcsinfo.pixel_shape`? If so, can we standardize and use that name and format?

---

I think so, yes. Happy to rename to `pixel_shape`. Would prefer not to put it in `wcsinfo` since there's not really WCS info in this schema.

---

Removed spatial_dimensions entirely, since this information can be accessed by getting the dimensions of the ndarray.